### DISCUSSION TITLE
Método de Pago

### DISCUSSION THREAD

**Discussion by GupySR** [Launch-X-Latam/MisionFrontEnd#433] [Team Posts]
Hola Muy buenas compañeros :), alguien me podría decir como puedo implementar métodos de pago en una pagina web de código abierto??

**Comment by VictorFalconi** [comment]
Hola. Podrias implementar checkout pro de mercado pago. Es realtivamente facil de implementar y tiene sandbox para hacer pruebas gratis

**Comment by Yhor93** [comment]
compañeros buenas noches, por factores varios no he podido ingresar a
clases, ya hice el proceso de team del on board pero aun desconozco como
puedo entrar a un grupo o una clase, alguien podría echarme una mano ?

**Comment by VictorFalconi** [comment]
Hola, compañero. Lo que tienes que hacer para entrar a las clases es descargar microsoft teams para la escuela o trabajo. Es diferente al microsoft teams para casa o pequeña empresa, en ese no vas a poder iniciar sesion. Para iniciar sesion te deben haber mandado un mail desde soporte con un correo que termina con @innovacion.mx y la contraseña. Accedes al microsoft teams con esas credenciales y en la parte izquierda puedes seleccionar la pestaña equipos. Ahi estara el equipo de Launch X, y ahi estan todas las clases y los links de los live que se van realizando

**Comment by Yhor93** [comment]
Pero en equipos no veo el de Launch X
Están:
UTVM
Alimentos para una dieta
Moda de Bordados
BUND/IKIGAI
Crew
Chat bot
Etc.....

bueno me comunicare por soporte a ver que me dicen, que estoy haciendo mal

**Comment by lucia123lu** [comment]
Hola, buenas, quería consultar sobre como entrar al equipo de launch x, ya que al entrar con email que me enviaron ustedes sigue sin aparecerme todos los links de las clases. Agradezco la ayuda
________________________________
De: Yhor Pabon ***@***.***>
Enviado: jueves, 6 de octubre de 2022 22:12
Cc: Subscribed ***@***.***>
Asunto: Re: [Launch-X-Latam/mision-frontend] Método de Pago (#2)


Pero en equipos no veo el de Launch X
Están:
UTVM
Alimentos para una dieta
Moda de Bordados
BUND/IKIGAI
Crew
Chat bot
Etc.....

bueno me comunicare por soporte a ver que me dicen, que estoy haciendo mal

**Comment by gumodi35** [comment]
Si aun no les aparece es que falta que les den el acceso, esperen un poco quizá mañana ya tengan el acceso, importante que entren con su cuenta de innovación para que les de el acceso.

**Comment by lucia123lu** [comment]
Perfecto, muchas gracias

________________________________
De: Gualberto ***@***.***>
Enviado: jueves, 6 de octubre de 2022 23:39
Cc: Lucía Stugelmayer ***@***.***>; Comment ***@***.***>
Asunto: Re: [Launch-X-Latam/mision-frontend] Método de Pago (#2)


Si aun no les aparece es que falta que les den el acceso, esperen un poco quizá mañana ya tengan el acceso, importante que entren con su cuenta de innovación para que les de el acceso.

**Comment by NoloMorales** [comment]
Muy bien, muchas gracias.
________________________________
De: Gualberto ***@***.***>
Enviado: jueves, 6 de octubre de 2022 09:41 p. m.
CC: Subscribed ***@***.***>
Asunto: Re: [Launch-X-Latam/mision-frontend] Método de Pago (#2)


Si aun no les aparece es que falta que les den el acceso, esperen un poco quizá mañana ya tengan el acceso, importante que entren con su cuenta de innovación para que les de el acceso.

**Comment by BushiMx** [comment]
> Si aun no les aparece es que falta que les den el acceso, esperen un poco quizá mañana ya tengan el acceso, importante que entren con su cuenta de innovación para que les de el acceso.

Muchas gracias por la recomendación. Saludos !

**Comment by sudoagile** [comment]
Igual yo no veo nada en el teams
